config/routes.rb

```diff
@@ -44,7 +42,6 @@
   post '/api/service_provider' => 'service_provider#update'
   post '/api/verify/images' => 'idv/image_uploads#create'
   post '/api/logger' => 'frontend_log#create'
-  post '/api/addresses' => 'idv/in_person/address_search#index'
```

Comment:
Since the AC said to leave this React component in place, I judged this update to be out-of-scope.

Reply:
Personally I'd rather us having removed the unused code and rely on git history if/when we need it in the future, since requirements change and we otherwise have no obligation to clean this up.